src/core/errorHandler.js

```diff
@@ -31,41 +36,63 @@ export class ErrorHandler {
 
 class ErrorHandlerComponent extends React.Component {
   static propTypes = {
+    autoRenderErrors: PropTypes.boolean,
     dispatch: PropTypes.func,
     error: PropTypes.object,
     errorHandlerId: PropTypes.string,
     WrappedComponent: PropTypes.object,
   }
 
+  static defaultProps = {
+    autoRenderErrors: true,
+  }
+
   render() {
     const {
+      autoRenderErrors,
       WrappedComponent,
       errorHandlerId,
       dispatch,
       error,
       ...props
     } = this.props;
-    const errorHandler = new ErrorHandler({ id: errorHandlerId, dispatch });
-    const wrappedOutput = (
-      <WrappedComponent errorHandler={errorHandler} {...props} />
-    );
 
+    let errorContent;
     if (error) {
-      return (
-        <div>
-          <ul className="ErrorHandler-list">
-            {error.messages.map((msg) => <li>{msg}</li>)}
-          </ul>
-          {wrappedOutput}
-        </div>
+      errorContent = (
```

Comment:
I think you could move this inside of `ErrorHandler` to put the logic closer to `getError()`.